Dwarf.h

```diff
@@ -1,7 +1,21 @@
 #pragma once
+#include "Character.h"
 
-struct Dwarf : Character
+struct Dwarf : public Character
 {
 private:
     const std::string name;
-};
\ No newline at end of file
+
+public:
+    // Constructor
+    Dwarf(std::string name_, int hp_, int armor_, int attackDamage_ = 4);
+
+    // Virtual Destructor
+    virtual ~Dwarf() override;
+
+    // Override the pure virtual function named getName
+    virtual const std::string& getName() override;
+
+    // Override the pure virtual function named getStats
+    virtual std::string getStats() override;
+};
```

Comment:
```
 10) correct any unnecessary keywords related to access specifiers used in the UDTs.
```
`struct` is public by default
put public things at the top, private things at the bottom.
- [ ] revise